hoot-core-test/src/test/cpp/hoot/core/schema/ScoreMatrixTest.cpp

```diff
@@ -73,6 +75,51 @@ class ScoreMatrixTest : public CppUnit::TestFixture
     uut.set(4, 2, .99);
 
     HOOT_STR_EQUALS(1.6, uut.minSumScore());
+
+    uut.resize(5, 3);
+
+    uut(0, 1) = 1;
+    uut(0, 2) = 0.8;
+
+    uut(1, 0) = .8;
+    uut(1, 1) = .98;
+    uut(1, 2) = 0.0;
+
+    uut(2, 0) = 1;
+    uut(2, 1) = .1;
+    uut(2, 2) = .98;
+
+    uut(3, 0) = .5;
+    uut(3, 1) = 1;
+    uut(3, 2) = 1;
+
+    uut(4, 0) = 1;
+    uut(4, 1) = .8;
+    uut(4, 2) = .99;
+
+    HOOT_STR_EQUALS(1.6, uut.minSumScore());
```

Comment:
This test checks the `operator()` that I created to make sure that it returns the correct reference to modify the value.